budgetportal/models.py

```diff
@@ -439,6 +443,8 @@ def get_programme_budgets(self):
             model_url,
             model_result.elapsed.microseconds / 1000
         )
+        if model_result.status_code == 404:
```

Comment:
Perhaps this should actually be allowed to result in exception and be considered a bug

Reply:
no this is fine for now - we'll notice very quickly that we're missing this data. And being able to generate the site lets us test things so long